monkey/infection_monkey/dataclasses/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .exploiter_result_data import ExploiterResultData
```

Comment:
We should call this directory something else. Open to suggestions.

Reply:
Any reason they can't just stay in `i_puppet`?

Reply:
`i_puppet.py` imports `TargetHost` which imports `PortScanData` from `i_puppet.py`.

Reply:
Sounds like we should move `TargetHost` to `i_puppet`.